allennlp/data/dataset_readers/conll2003.py

```diff
@@ -56,22 +57,29 @@ class Conll2003DatasetReader(DatasetReader):
         Each will have its own namespace: ``pos_labels``, ``chunk_labels``, ``ner_labels``.
         If you want to use one of the labels as a `feature` in your model, it should be
         specified here.
+    coding_scheme: ``str``, optional (default=``iob1``)
+        Specifies the coding scheme for ``ner_labels`` and ``chunk_labels``.
+        Valid options are ``iob1`` and ``biolu``.
```

Comment:
What does the 1 mean here? Typically i've referred to (I think the same scheme) as BIO tags, it would be good to be consistent

Reply:
Yeah e.g in the CRF tagger we refer to these as BIO and BIOUL, could you just match that:
https://github.com/allenai/allennlp/blob/master/allennlp/models/crf_tagger.py#L36

Reply:
I've seen these referred to all sorts of different names in various places.  IOB1 is distinct from IOB2 so I wanted to be explicit about the BIO tagging scheme we were using.  IOB1 only uses B when it follows an I of the same type, otherwise entity spans start with I.  IOB2 always uses B to start the entity span.

Reply:
> What does the 1 mean here? Typically i've referred to (I think the same scheme) as BIO tags, it would be good to be consistent

Hi~
I'm still confused about the BIO1,BIO2 and BIOUL. Could you please explain it?
Thanks a lot.